core/frontend/src/components/health/GpsTrayMenu.vue

```diff
@@ -40,6 +98,85 @@ export default Vue.extend({
     }
   },
   computed: {
+    values():
+      Array<
+        {
+          name: string,
+          value: string,
+          tooltip?: string,
+          icon: string,
+          link?: string,
+          link_icon?: string
+        }> {
+      if (!this.gps_raw_int) {
+        return []
+      }
+      const values = [
+        {
+          name: 'Position',
+          value: `${this.lattitude.toFixed(2)}º / ${this.longitude.toFixed(2)}º`,
+          tooltip: 'Position coordinates',
+          icon: 'mdi-map-marker',
+          link: this.map_link,
+          link_icon: 'mdi-map-search',
+        },
+        {
+          name: 'Altitude',
+          value: `${(this.gps_raw_int.alt / 1000).toFixed(2)} m`,
+          tooltip: 'Altitude in meters',
+          icon: 'mdi-arrow-collapse-up',
+        },
+        {
+          name: 'Satellites',
+          value: `${this.satellites_number}`,
+          tooltip: 'Number of satellites',
+          icon: 'mdi-satellite-variant',
+        },
+        {
+          name: 'Fix',
+          value: `${this.connection_description}`,
+          tooltip: 'GPS connection status',
+          icon: 'mdi-crosshairs-gps',
+        },
+        {
+          name: 'HDOP',
+          value: `${(this.gps_raw_int.eph / 100).toFixed(2)}`,
```

Comment:
Could this be augmented with a [qualitative interpretation](https://en.wikipedia.org/wiki/Dilution_of_precision_(navigation)#Interpretation) (e.g. `Excellent (1.7)`)? That seems more useful/understandable than just a magic number.

Reply:
That is kind of hard. This qualitative values can change for user to user and different use cases. This can also result in increased support, as we had for the network interface speed test.

Reply:
Fair enough.

We could make it configurable, but that's probably too complex to be worth it, and too much scope creep for this PR.